### PR TITLE
Fix example to check if an item is the site front page

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: develop/plone/serving/traversing.rst [hunk 1]
```diff
@@ -445,24 +445,17 @@ for the default item) in page templates:
...
-Example code below::
+Example:
 
-    from zope.component import getMultiAdapter
-    from plone.app.layout.navigation.interfaces import INavigationRoot
-
-    def isFrontPage(self):
-        """
-        Check if the viewlet is on a front page.
+.. code:: python
```
Comment (svx) [inline develop/plone/serving/traversing.rst]: needs an empty line here :)

**Comment by svx** [comment]
Thank you !

**Comment by hvelarde** [comment]
do you want me to fix this in the other branches as well?

**Comment by svx** [comment]
If you would like to do so, sure ! :)

**Pull request merged**